test/e2e/node/kubelet.go

```diff
@@ -409,10 +409,7 @@ var _ = SIGDescribe("kubelet", func() {
 			})
 
 			ginkgo.AfterEach(func(ctx context.Context) {
-				err := e2epod.DeletePodWithWait(ctx, c, pod)
-				framework.ExpectNoError(err, "AfterEach: Failed to delete client pod ", pod.Name)
-				err = e2epod.DeletePodWithWait(ctx, c, nfsServerPod)
-				framework.ExpectNoError(err, "AfterEach: Failed to delete server pod ", nfsServerPod.Name)
+				e2epod.DeletePodsWithWaitByBatch(ctx, c, []*v1.Pod{pod, nfsServerPod})
```

Comment:
As tests usually own pods, would it make sense to ask api server to delete them at once, e.g. by labelling them and using `DeleteCollectioin` with `LabelSelector`?